indico/modules/categories/client/js/components/CategoryStatistics.jsx

```diff
@@ -0,0 +1,211 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import categoryStatisticsURL from 'indico-url:categories.statistics_json';
+
+import moment from 'moment';
+import React, {useCallback, useMemo} from 'react';
+import PropTypes from 'prop-types';
+import {Chart} from 'react-charts';
+import {Container, Header, Segment, Statistic, Message, Loader} from 'semantic-ui-react';
+import {Translate} from 'indico/react/i18n';
+
+import {useIndicoAxios} from 'indico/react/hooks';
+
+export default function CategoryStatistics({categoryId, lang}) {
+  const {data, loading: isLoading} = useIndicoAxios({
+    url: categoryStatisticsURL({category_id: categoryId}),
+    camelize: true,
+    trigger: categoryId,
+  });
+
+  if (isLoading) {
+    return <Loader size="massive" active />;
+  } else if (!data) {
+    return null;
+  }
+
+  return <CategoryStatisticsDisplay data={data} lang={lang} />;
+}
+
+CategoryStatistics.propTypes = {
+  categoryId: PropTypes.number.isRequired,
+  lang: PropTypes.string.isRequired,
+};
+
+function CategoryStatisticsDisplay({data, lang}) {
+  const userLanguage = lang.replace('_', '-');
+  const updatedTime = moment.utc(data.updated).format('D MMM YYYY HH:mm');
```

Comment:
This format will not take into account the current locale.. use `L` or something similar ;) and also take a look at `serializeDate`

Reply:
Can we show time as well by `serializeDate`?

Reply:
By using `LL`.

![Screenshot from 2020-03-09 12-20-21](https://user-images.githubusercontent.com/24358501/76208787-ad958200-6200-11ea-9dfc-ab7bddff9d60.png)

By using `L`:
![Screenshot from 2020-03-09 12-21-00](https://user-images.githubusercontent.com/24358501/76208804-b5552680-6200-11ea-9d8f-0ecefa4c19af.png)

we need to also show time as well.

Reply:
Just check the momentjs docs for locale-aware formats :)

Reply:
and yeah, you can show times with `serializeDate` as well.. it will use whatever format you pass to it ;)

Reply:
okay, I can set the locale using [`moment.locale(userLanguage)`](https://momentjs.com/docs/#/i18n/) and then instead of formatting the date as `'D MMM YYYY HH:mm'`, I can pass just the format as `L`, `LL`, `LLL`. Now the time and date both are visible. Resolving this. :)